app/reported/page.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { cookies } from 'next/headers';
+import { redirect } from 'next/navigation';
+
+import Posts from '@/app/_components/posts';
+import config from '@/config.json';
+import { prisma } from '@/utils/prisma';
+import Confirmation from './confirmation';
+
+const Reported = async () => {
```

Comment:
```suggestion
export default async function Reported() {
```

oder kp wie die Reihenfolge ist :D